src/internal/m365/service/sharepoint/export.go

```diff
@@ -72,30 +73,49 @@ func (h *baseSharePointHandler) ProduceExportCollections(
 	)
 
 	for _, dc := range dcs {
-		drivePath, err := path.ToDrivePath(dc.FullPath())
-		if err != nil {
-			return nil, clues.WrapWC(ctx, err, "transforming path to drive path")
+		cat := dc.FullPath().Category()
+
```

Comment:
Since you're adding it to the error returns anyway.
```suggestion
		cat := dc.FullPath().Category()
                ictx := clues.Add(ctx, "fullpath_category", cat)

```